### ISSUE TITLE
Refused to display [url] in a frame because it set 'X-Frame-Options' to 'deny'

### ISSUE THREAD

**Issue by jedabero** [jeduan/cordova-plugin-facebook4#546]
I've having this issue when serving my app by browser. 
```
Refused to display 'https://www.facebook.com/login.php?skip_api_login=1&api_key=bunch_of_more_parameters' in a frame because it set 'X-Frame-Options' to 'deny'.
InAppBrowserProxy.js:42 Uncaught DOMException: Blocked a frame with origin "https://[mydomainredacted]" from accessing a cross-origin frame.
    at HTMLIFrameElement.<anonymous> (https://[mydomainredacted]/plugins/cordova-plugin-inappbrowser/src/browser/InAppBrowserProxy.js:42:70)
```
This happens when calling
```js
facebookConnectPlugin.login(['public_profile', 'user_friends', 'email'],
  facebookLoginSuccess,
  function (error) {
    f7.addNotification({
      title: 'Error',
      message: errorMessage + ' : ' + JSON.stringify(error)
    })
  }
);
```
`facebookLoginSuccess` does not get called nor the error function

If I open the link in the first error it shows the facebook login page (because I'm not logged in)

I've configured my facebook app correctly by adding my domain to App Domains as [shown (instead of localhost, I've put my domain](https://github.com/jeduan/cordova-plugin-facebook4/blob/master/docs/browser/README.md)

My app is working properly on iOS and Android

I'm serving the www folder in platforms/browser after building it by `phonegap build browser`

**Comment by ThorvaldAagaard** [comment]
Could it be that this is a Firefox only issue?

**Comment by rahulpsd18** [comment]
Not working on chrome either. 
Getting `Refused to display 'https://m.facebook.com/v2.7/dialog/oauth?app_id=...&...&sdk=joey&version=v2.7' in a frame because it set 'X-Frame-Options' to 'deny'. ` in console.

**Comment by HardikDG** [comment]
Any solution for this? I am also facing this issue.

**Comment by peterpeterparker** [comment]
Maybe this is the solution https://stackoverflow.com/a/26143574/5404186 ?

Feel free to try and if it works to submit a PR

**Comment by ThorvaldAagaard** [comment]
Now Chrome also have this error.

Can bee seen on https://geme-cce67.firebaseapp.com

As far as I know there has been no changes in the App
![image](https://user-images.githubusercontent.com/10560652/40551064-37c285fe-603c-11e8-803c-6de8f0c62659.png)

And the FB-login, looks ok to me

![image](https://user-images.githubusercontent.com/10560652/40551550-9c7b559c-603d-11e8-8aaf-b2e88730c6d7.png)


and the wesite setup
![image](https://user-images.githubusercontent.com/10560652/40551590-b668d056-603d-11e8-972d-7362f211ee16.png)

**Comment by lucasblackhurst** [comment]
I'm receiving the exact same issue as of today. Any news?

**Comment by FREEZX** [comment]
Try removing the cordova-plugin-inappbrowser plugin.
Seems like that was causing the issues for me.

**Comment by exequielc** [comment]
@lucasblackhurst  same problem too

**Comment by manzajaro** [comment]
+1

**Comment by iburrows** [comment]
I am not using cordova and I still have this issue

**Comment by FREEZX** [comment]
> I am not using cordova and I still have this issue

If you're using a cordova plugin, whatever you're using probably uses cordova in the background

**Comment by listentotherainbow** [comment]
Debugging the Facebook sdk i have found the problem seems to be that when inAppBrowser is installed the Facebook sdk detects that we are on a mobile phone even when running from a browser. (I assume inAppBrowser is responsbible for the change of navigator.UserAgent that happens)

Using Chrome Developer tools to force the navigator.UserAgent to be fixed to the Chrome UserAgent in fact made all work, but that is of course not a proper solution

As I need inAppBrowser to open legal documents etc I could not just remove it, but the is a way to avoid using inAppBrowser when on a browser or probably also only when performing tasks like login

At the top of my app before loading any plugins you can save window.open like this

    var original_open = window.open;

then later before you active login or if you want to after detecting we are on a browser you can do something like

      if (!this.isApp) {
        window.open = original_open;
      }

So basically the error message is because the platform is being detected wrongly

**Comment by noahcooper** [comment]
This plugin is deprecated. Check out `cordova-plugin-facebook-connect` at https://www.npmjs.com/package/cordova-plugin-facebook-connect. 

Since this issue was logged, `cordova-plugin-inappbrowser` no longer clobbers `window.open`, and Facebook Login works as expected on the browser platform.

**Comment by IAMPHARO** [comment]
I have alot of these issues
